modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,246 @@
+
+* This update supports advanced event matching on file path for push and pull requests to GitHub and GitLab. For example, you can use the Common Expression Language (CEL) to run a pipeline only if a path has changed and match each markdown file only in the `docs` directory. 
++
+[source,yaml]
+----
+  ...
+  annotations:
+     pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "pull_request" && "docs/*.md".pathChanged()
+----
+
+* This update adds a new driver to support the Gitea software package.
+
+* With this update, you can reference a remote pipeline in the `pipelineRef:` object using annotations.
+
+* With this update, you can auto-configure new GitHub repositories with {pac}, which sets up a namespace and creates a custom resource for your repository.
+
+* This update provides metrics for pipeline runs with provider information in the user namespace.
+
+* This update provides the following enhancements for the `tkn-pac` plugin:
+** Detects running pipelines correctly 
+** Fixes showing duration when there is no failure completion time
+** Shows an error snippet and highlights the error regular expression pattern in the `tkn-pac describe` command
+** Adds the `use-real-time` switch to the `tkn-pac ls` and `tkn-pac describe` commands
+** Imports the `tkn-pac` logs documentation
+** Shows `pipelineruntimeout` as a failure in the `tkn-pac ls` and `tkn-pac describe` commands.
+** Show a specific pipeline run failure with the `--target-pipelinerun` option.
+      
+* With this update, you can view the errors for your pipeline run in the form of a version control system (VCS) comment or a small snippet in the GitHub checks. 
+
+* With this update, {pac} optionally can detect errors inside the tasks if they are of a simple format and add those tasks as annotations in GitHub. This update is part of an alpha preview feature.
+
+* This update adds the following new commands:
+** `tkn-pac webhook add`: Adds a webhook to project repository settings and updates the `webhook.secret` key in the existing `k8s Secret` object without updating the repository. 
+** `tkn-pac webhook update-token`: Updates provider token for an existing `k8s Secret` object without updating the repository.
+
+* This update enhances functionality of the `tkn-pac create repo` command, which creates and configures webhooks for GitHub, GitLab, and BitbucketCloud along with creating repositories. 
+
+* With this update, the `tkn-pac describe` command shows the latest fifty events in a sorted order.
+
+* This update adds the `--last` option to the `tkn-pac logs` command.
+
+* With this update, the `tkn-pac resolve` command prompts for a token on detecting a `git_auth_secret` in the file template.
+
+* With this update, {pac} hides secrets from log snippets to avoid exposing secrets during a pipeline run.
```

Comment:
We should clarify that it's about log snippets in GitHub UI, not pipeline run logs

Reply:
Incorporated. Please have a look at the updated RN.